cmd/plugins/main.go

```diff
@@ -87,9 +86,11 @@ func main() {
 	}
 
 	s := grpc.NewServer()
-
 	extProcPb.RegisterExternalProcessorServer(s, gateway.NewServer(redisClient, k8sClient))
-	healthPb.RegisterHealthServer(s, gateway.NewHealthCheckServer())
+
+	healthCheck := health.NewServer()
```

Comment:
this server automatically implements `Check` and `Watch` so we do not need maintain healthcheck server ourselves?